lib/network.py

```diff
@@ -1059,27 +1034,132 @@ def follow_chain(self, index):
     def get_local_height(self):
         return self.blockchain().height()
 
-    def synchronous_get(self, request, timeout=30):
+    def __wait_for(it):
```

Comment:
Even if it's not needed, could you make this a `@staticmethod`?

Reply:
This is indeed not needed but I'll gladly conform.